app/views/stats/index.html.erb

```diff
@@ -24,12 +24,18 @@
 <div class="wrapper">
   <dl class="statistics meta group" role="complementary">
     <% [:user_subscriptions, :kudos, :comments, :bookmarks, :subscriptions, :word_count].each do |stat| %>
-      <dt><%= (stat.to_s.titleize == "Comments") ? ts("Comment Threads") : stat.to_s.titleize %></dt>
-      <dd><%= @totals[stat] %></dd>
+      <% if stat == :word_count
+          html_class = 'words'
+        else
+          html_class = stat.to_s.humanize.downcase
+        end
+      %>
+      <dt class="<%= html_class %>"><%= (stat == :comments) ? ts("Comment Threads:") : stat.to_s.titleize %></dt>
```

Comment:
But that lovely succinct code :crying_cat_face: 
I suppose you could possibly change the list into a hash and use the key for the CSS class and the value as the display text (though this hinges on whether the loop would work with `ts()`):

```ruby
{ user_subscriptions: ts("User subscriptions"), 
kudos: ts("Kudos"), 
comments: ts("Comment threads"), 
bookmarks: ts("Bookmarks"), 
subscriptions: ts("Subscriptions"), 
words: ts("Word count") } each do |stats, stat_text|
```